### DISCUSSION TITLE
vscode+msys2+vcpkg+cmake

### DISCUSSION THREAD

**Discussion by raspberrypisig** [libtcod/libtcod#103] [Show and tell]
I hope you don't mind, but it took me a while to get libtcod to work for the x64-mingw-static triplet to work on Windows 11, so I would like to share the config that finally worked for me.

https://github.com/raspberrypisig/roguelike-firststab/tree/setup

**Comment by HexDecimal** [comment]
This is more an issue with how difficult it is to setup a MinGW tool chain on Windows rather than an issue with libtcod itself.  I'll always recommend using MSVC to Windows devs since it's more accessible, and I really don't want to troubleshoot the more obscure problems with MinGW that would come up if I recommended it.

I've moved this to the discussions board in case that makes it more visible to anyone that's interested.
